source/Conversation.h

```diff
@@ -33,6 +33,17 @@ class Sprite;
 // killed. A conversation can also branch based on various condition flags that
 // are set for the player, or even trigger various changes to the game's state.
 class Conversation {
+	class Data {
```

Comment:
```suggestion
public:
	class Data {
```

Reply:
`Data` doesn't need to be a public class that all other classes can see and use.

Instead, it should be moved below the public methods and be explicitly declared private.